paraatm/tests/IFHM_sim_example.py

```diff
@@ -0,0 +1,224 @@
+"""
+NASA University Leadership Initiative program
+Information fusion for real-time national air transportation system prognostics under uncertainty
+
+    Real-time flight health monitoring uisng autoencoder (AE)
+    - From KSFO to KPHX during cruise to approach operation
+    - Generate normal & upset flight scenarios
+    - Flight health monitoring & upset detection
+
+    @author: Hyunseong Lee, Adaptive Intelligent Materials & Systems (AIMS) Center,
+             Arizona State University
+    
+    Last modified on 5/25/2020
+"""
+
+# Locate PARA_ATM Home directory
+import os
+os.chdir("/home/hector/Documents/para-atm-master")
```

Comment:
You'll need to remove all references to your local filesystem.  para-atm itself does not need to be run from any particular directory.  If you are having trouble loading sample files that are stored inside para-atm, I can help with that.